docs/csharp/language-reference/tokens/toc.md

```diff
@@ -0,0 +1,3 @@
+# [C# Special Characters](index.md)
+## [$](interpolated.md)
```

Comment:
I've modified the TOC -- I think that that at least makes it clear what the symbols are called.